### PR TITLE
feat: Created Readme

### LINKED ISSUE #1: Readme
Please, add a readme so we can know what the app is for!
Looking forward to contribute 👯 
Thanks,
Tom

### PR DESCRIPTION
**Pull request by quentin-sommer**
fixes #1 

I didn't find any code of conduct to sign, will the steering committee still consider this pull request? I've put a great deal of work into it.

Best

Quentin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# sport-max
+
+
+The maximum of sport
```
Comment (tmos) [inline README.md]: Where the heck is the final point ? Please, add it before merging.

Comment (quentin-sommer) [reply README.md]: Good catch! fixed. Thank you.

**Comment by quentin-sommer** [comment]
Does this PR look good to you @Spieldy? Really looking forward to this getting merged. My project depends on this feature.

**Comment by quentin-sommer** [comment]
@tmos @Spieldy Since this repo magically reappeared should we continue forward with this or can I close it?

**Pull request closed without merging**